_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -21,39 +21,33 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is a set of one or more [space separated tokens](https://html.spec.whatwg.org/#set-of-space-separated-tokens), except if one of the following is true:
+The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` [attribute value][] that is neither empty (`""`) nor only [ASCII whitespace][], except if one of the following is true:
 
-- The element is not [visible](#visible), and not [included in the accessibility tree](#included-in-the-accessibility-tree)
-- The element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`
-- The element has an `aria-disabled` [attribute value][] of `true`
-- The element is not part of [sequential focus navigation](https://html.spec.whatwg.org/#sequential-focus-navigation) and has a [semantic role](#semantic-role) that is not a [widget role](https://www.w3.org/TR/wai-aria-1.1/#widget_roles).
+- The element is not [visible][], and not [included in the accessibility tree][]; or
+- The element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`; or
+- The element has an `aria-disabled` [attribute value][] of `true`; or
+- The element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][].
```

Comment:
```suggestion
- (**not perceivable**) the element is not [visible][], and not [included in the accessibility tree][]; or
- (**inactive**) the element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`; or
- (**disabled**) the element has an `aria-disabled` [attribute value][] of `true`; or
- (**not in focus order**) the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][].
```

Reply:
Did this slightly differently, but good idea.